statistics/table.go

```diff
@@ -299,6 +283,24 @@ func (t *Table) String() string {
 	return strings.Join(strs, "\n")
 }
 
+func (t *Table) indexStartWithColumnForDebugLog(colName string) *Index {
```

Comment:
```
func (t *Table) indexStartWithColumnForDebugLog(colName string) *Index, bool 
```

How about this? We can fast check whether `index`  is for debug purpose or not. If not, we can simply return `nil, false`.

Reply:
You mean that check the log level here?

Reply:
Sorry for the delay. Yes, we can check the log level here.

Reply:
But it seems ok currently～